Index: wget-1.25.0/build-aux/config.sub
===================================================================
--- wget-1.25.0.orig/build-aux/config.sub
+++ wget-1.25.0/build-aux/config.sub
@@ -180,7 +180,7 @@ case $1 in
 						basic_os=$field2
 						;;
 					# Manufacturers
-					dec* | mips* | sequent* | encore* | pc533* | sgi* | sony* \
+					dec* | mips* | sw*| sequent* | encore* | pc533* | sgi* | sony* \
 					| att* | 7300* | 3300* | delta* | motorola* | sun[234]* \
 					| unicom* | ibm* | next | hp | isi* | apollo | altos* \
 					| convergent* | ncr* | news | 32* | 3600* | 3100* \
@@ -1211,6 +1211,7 @@ case $cpu-$vendor in
 			| le32 | le64 \
 			| lm32 \
 			| loongarch32 | loongarch64 \
+                        | sw_64 \
 			| m32c | m32r | m32rle \
 			| m5200 | m68000 | m680[012346]0 | m68360 | m683?2 | m68k \
 			| m6811 | m68hc11 | m6812 | m68hc12 | m68hcs12x \
